dashboard/app/views/levels/_admin.html.haml

```diff
@@ -8,7 +8,7 @@
       %li= link_to "see callouts (#{@level.available_callouts(@script_level).count})", show_callouts: '1'
       - if @level.is_a? Gamelab
         %li= link_to 'view animation JSON', '', onclick: 'window.viewExportableAnimationList(); return false'
-      - if Rails.application.config.levelbuilder_mode
```

Comment:
I copied this param from elsewhere in this file: https://github.com/code-dot-org/code-dot-org/blob/b1c53b7a0ab9f5aa3e90ad7dd6a77b5f2bc7454e/dashboard/app/views/levels/_admin.html.haml#L111

Reply:
also, this is where the channel id param is coming from: https://github.com/code-dot-org/code-dot-org/blob/e5125d0fd4c1d575a33b99dc8b7e9995bf4af532/dashboard/config/routes.rb#L247

which helps explain why it won't be present on script levels.

Reply:
Thanks for doing this! Wondering if it would be clearer just to check explicitly whether the level name is one of our standalone project levels, like we do here (looks like you put in a guard against changing level names of standalone project levels a while back 👏 ):

https://github.com/code-dot-org/code-dot-org/blob/a6eb20471940fd7e4a28bef4384874d14c83d2d9/dashboard/app/controllers/levels_controller.rb#L160

Reply:
haha, nice find!

If we do this the same way, then after clicking through to the level page, you would still not be able to edit the level, effectively making it so that no aspect of standalone project levels can be edited using levelbuilder UI, either locally or on levelbuilder. So since the behavior is different that way, I would say that we should decide on which behavior we want first, and then decide the simplest way to implement it.

In case we want to keep the behavior implemented in this PR, I've added a comment to help explain what's going on. It is still a bit loose because it relies on the assumption that level pages never set the channel id param. Do you think it would help to find a way to test more directly that we are on a route managed by the projects controller?

Reply:
tbh I still get turned around in the world of standalone project levels, so if you think what you've got here is the most straightforward way to handle this, I'm down with it :)

Reply:
Good point! A higher hurdle does seem better and that workaround seems sufficient. I'll update

Reply:
Thanks for suggesting this Ben! Verified as follows:

* ![Screenshot 2024-01-08 at 1 55 48 PM](https://github.com/code-dot-org/code-dot-org/assets/8001765/a8b32a4d-7be4-4bbc-8f2e-c4d91b776220)

* ![Screenshot 2024-01-08 at 1 56 05 PM](https://github.com/code-dot-org/code-dot-org/assets/8001765/30d01564-2223-4de9-814d-725207f76569)

* ![Screenshot 2024-01-08 at 1 56 29 PM](https://github.com/code-dot-org/code-dot-org/assets/8001765/d97b426f-41d7-4d25-aeab-c3bf979dacee)